chatty-clownfish/bangazon-sprint-2
# Rh linkstoproductlist

## Link to Ticket
https://github.com/chatty-clownfish/bangazon-sprint-2/issues/36
Closes # [Link]()
Completed: Ticket #13 

## Description of Proposed Changes

-Hyperlinks added to homepage that lead to individual product type listings
-Login authentication added to sell products tab and is ready for other pages that require login
-Register button added to login page

## Steps to Test

-Check for working register button on login page
-Check authentication on sell tab in navbar when not logged in -- should lead to login page, and after logging in, should lead to sell item page. Same with 'my profile' page. This should work for all pages that require login once we add decorators
-Check for working hyperlinks on home/categories page

Outline the steps to test

```sh
git fetch --all
git checkout rh-linkstoproductlist
```

## Impacted Areas in Application

List general components of the application that this PR will affect:
Home page
Login Page
General authentication
Navbar (removed hiding of sell and my profile buttons)
* 

## Mentions @username

Tag users that need to review this code
